db/migrate/20151125195635_acts_as_taggable_on_migration.acts_as_taggable_on_engine.rb

```diff
@@ -1,31 +1,35 @@
 # This migration comes from acts_as_taggable_on_engine (originally 1)
 class ActsAsTaggableOnMigration < ActiveRecord::Migration[4.2]
   def self.up
-    unless ActiveRecord::Base.connection.table_exists?('tags')
-      create_table :tags do |t|
-        t.string :name
-      end
-    end
+    self.create_tags
+    self.create_taggings
+  end
 
-    unless ActiveRecord::Base.connection.table_exists?('taggings')
-      create_table :taggings do |t|
-        t.references :tag
+  def self.create_tags
+    return if ActiveRecord::Base.connection.table_exists?('tags')
+    create_table :tags do |t|
+      t.string :name
+    end
+  end
 
-        # You should make sure that the column created is
-        # long enough to store the required class names.
-        t.references :taggable, polymorphic: true
-        t.references :tagger, polymorphic: true
+  def self.create_taggings
+    return if ActiveRecord::Base.connection.table_exists?('taggings')
+    create_table :taggings do |t|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->